CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 - isArray validator can now validate items with validators and multiple types.
+- `@Controller` and all route decorators do accept `RequestHandler` as middlewares.
```

Comment:
`@Controller` and `@Route` decorators do accept `RequestHandler` as middlewares.